test/index.test.js

```diff
@@ -3,120 +3,174 @@
+      let message = logs.shift();
+      expect(message.level).to.be.eql(50);
 
-    expect(message.a).to.be.eql(1);
+      // Should keep the object safe:
+      logger.error({
+        a: 1
+      }, 'hello');
+      message = logs.shift();
 
-    // Should use the Error object for the stack:
-    logger.error(new Error('Fatal error'), 'hello');
-    message = logs.shift();
+      expect(message.a).to.be.eql(1);
 
-    expect(message.err).to.be.an('object');
-    expect(message.err.stack.substr(0, 18)).to.be.eql('Error: Fatal error');
-  });
-  it('should output the warn level message', () => {
-    logger.warn('hello');
-    const message = logs.shift();
-    expect(message).to.have.property('level', 40);
-    expect(message).to.not.have.property('stack');
-  });
-  it('should output the info level message', () => {
-    logger.info('hello');
-    const message = logs.shift();
-    expect(message).to.have.property('level', 30);
-  });
-  it('should output the debug level message', () => {
-    logger.debug('hello');
-    const message = logs.shift();
-    expect(message).to.have.property('level', 20);
-  });
-  it('should not output the trace level message (level set to debug)', () => {
-    logger.trace('hello');
-    expect(logs).to.have.lengthOf(0);
-  });
+      // Should use the Error object for the stack:
+      logger.error(new Error('Fatal error'), 'hello');
+      message = logs.shift();
 
-  it('should format message correctly', () => {
-    logger.info('hello %s', 20);
-    const message = logs.shift();
-    expect(message.msg).to.be.eql('hello 20');
-  });
+      expect(message.err).to.be.an('object');
+      expect(message.err.stack.substr(0, 18)).to.be.eql('Error: Fatal error');
+    });
+    it('should output the warn level message', () => {
+      logger.warn('hello');
+      const message = logs.shift();
+      expect(message).to.have.property('level', 40);
+      expect(message).to.not.have.property('stack');
+    });
+    it('should output the info level message', () => {
+      logger.info('hello');
+      const message = logs.shift();
+      expect(message).to.have.property('level', 30);
+    });
+    it('should output the debug level message', () => {
+      logger.debug('hello');
+      const message = logs.shift();
+      expect(message).to.have.property('level', 20);
+    });
+    it('should not output the trace level message (level set to debug)', () => {
+      logger.trace('hello');
+      expect(logs).to.have.lengthOf(0);
+    });
+
+    it('should format message correctly', () => {
+      logger.info('hello %s', 20);
+      const message = logs.shift();
+      expect(message.msg).to.be.eql('hello 20');
+    });
 
-  it('should keep objects safe', () => {
-    logger.info({
-      a: 1,
-      b: {
+    it('should keep objects safe', () => {
+      logger.info({
+        a: 1,
+        b: {
+          c: 1
+        }
+      }, 'hello');
+      const message = logs.shift();
+      expect(message.a).to.be.eql(1);
```

Comment:
There are more params in `message` which are not really interesting, such as `hostname`, `pid` or `time`